types/2022-08-01/Invoices.d.ts

```diff
@@ -1995,12 +1991,6 @@ declare module 'stripe' {
           phone?: string;
         }
 
-        namespace Shipping {
-          interface Address extends Omit<Stripe.AddressParam, 'line1'> {
```

Comment:
Should we leave aliases from the old type to the new type to avoid the "breaking" change?

```
type Address = Stripe.AddressParam
``` 

sort of thing

Reply:
We did it when moving errors around in https://github.com/stripe/stripe-node/pull/1375/files

Reply:
(discussed in slack)